components/data-display/member-card.tsx

```diff
@@ -125,6 +157,22 @@ export const MemberCard: FC<MemberCard> = ({
               </MenuList>
             </Menu>
           ) : undefined}
+
+          {/* 権限変更確認モーダル */}
+          <Modal isOpen={isOpen} onClose={onClose}>
+            <ModalOverlay />
+              <ModalHeader>権限変更の確認</ModalHeader>
+              <ModalBody>
+                {selectedMemberName} さんの役割を「{selectedCurrentRoleName}」から「{selectedRoleName}」に変更しますか？
+              </ModalBody>
+              <ModalFooter>
+                <Button onClick={onClose}>キャンセル</Button>
+                <Button colorScheme="blue" onClick={confirmRoleChange} ml={3}>
```

Comment:
`margin-left`は使わずに、`ModalFooter`の`gap`で間隔を空けてください。

```suggestion
                <Button colorScheme="blue" onClick={confirmRoleChange} >
```